geode-core/src/main/java/org/apache/geode/distributed/LocatorLauncher.java

```diff
@@ -1781,8 +1782,8 @@ protected void validate() {
      * @see org.apache.geode.distributed.LocatorLauncher.Command#START
      */
     protected void validateOnStart() {
-      if (Command.START.equals(getCommand())) {
-        if (StringUtils.isBlank(getMemberName())
+      if (Command.START == getCommand()) {
```

Comment:
Yep, IntelliJ has an automated refactoring and inspection for this. I don't even remember making this change unless I was trying to make LocatorLauncher more consistent with ServerLauncher.